### DISCUSSION TITLE
JanusVideoRoomPlugin: I am stream with JanusVideoRoomPlugin plugin when when stream is started to server local preview is also affected.

### DISCUSSION THREAD

**Discussion by ChiragLiveu** [flutterjanus/flutter_janus_client#163] [Q&A]
I am using the JanusVideoRoomPlugin for streaming from a mobile device. When the user launches the app, I show the local preview with the required constraints, and it displays properly. However, when I start the streaming from the mobile, sometimes the local preview is affected by low FPS and low resolution. Can you please help me understand why this happens to the local preview when I am in an active session?"

**Comment by shivanshtalwar** [comment]
Can you share a reproducible example of it
Thanks
S.T

**Comment by ChiragLiveu** [comment]
I have attached the file please check and let me know I am wrong anywhere.
Thanks you for your help
[Demo Code Janus.docx](https://github.com/flutterjanus/flutter_janus_client/files/15317375/Demo.Code.Janus.docx)

**Reply by ChiragLiveu** [reply]
Have you checked? Can you please checked and revert back to me ?
